CareKit/CareContents/OCKCareContentsViewController.h

```diff
@@ -229,6 +253,14 @@ OCK_CLASS_AVAILABLE
  */
 @property (nonatomic, nullable) NSString *readOnlySectionHeader;
 
+/**
+ A message that will be displayed in the table view's background view if there are
+ no intervention activities to display.
```

Comment:
Optional: A message that will be displayed in the table view's background view if there are no interventions, assessments, or ReadOnly activities to display.